Shared/Samples/Browse OGC API feature service/BrowseOGCAPIFeatureServiceView.swift

```diff
@@ -0,0 +1,260 @@
+        }()
+        
+        @Published var layerNames: [String] = []
+        /// The geometry that represents a rectangular shape that encompasses the area.
+        private(set) var completeExtent: Envelope?
+        private var featureCollectionInfos: [OGCFeatureCollectionInfo] = []
+        private var service: OGCFeatureService!
+        private(set) var selectedInfo: OGCFeatureCollectionInfo?
+        
+        /// The query parameters to populate features from the OGC API service.
+        private let queryParameters: QueryParameters = {
+            let queryParameters = QueryParameters()
+            // Set a limit of 1000 on the number of returned features per request,
+            // because the default on some services could be as low as 10.
+            queryParameters.maxFeatures = 1_000
+            return queryParameters
+        }()
+        
+        /// Returns a renderer for a specified geometry type.
+        /// - Parameter geometryType: The ARCGis Geometry type.
+        /// - Returns: Returns a `SimpleRenderer` optional with the correct settings for the given geometry.
+        private func getRenderer(withType geometryType: Geometry.Type) -> SimpleRenderer? {
+            var renderer: SimpleRenderer?
+            if geometryType == Point.self {
+                renderer = SimpleRenderer(
+                    symbol: SimpleMarkerSymbol(
+                        style: .circle,
+                        color: .blue,
+                        size: 5
+                    )
+                )
+            }
+            if geometryType == Multipoint.self {
+                renderer = SimpleRenderer(
+                    symbol: SimpleMarkerSymbol(
+                        style: .circle,
+                        color: .blue,
+                        size: 5
+                    )
+                )
+            }
+            if geometryType == Polyline.self {
+                renderer = SimpleRenderer(
+                    symbol: SimpleLineSymbol(
+                        style: .solid,
+                        color: .blue,
+                        width: 1
+                    )
+                )
+            }
+            if geometryType == Polygon.self {
+                renderer = SimpleRenderer(
+                    symbol: SimpleFillSymbol(
+                        style: .solid,
+                        color: .blue,
+                        outline: nil
+                    )
+                )
+            }
+            return renderer
+        }
+        
+        /// Create and load the OGC API features service from a URL.
+        /// - Parameter url: The URL of the OGC service.
+        /// - Returns: Returns a `OCGFeatureService` that has been loaded and initialized.
+        private func makeService(url: URL) async throws -> OGCFeatureService {
+            let service = OGCFeatureService(url: url)
+            try await service.load()
+            if service.loadStatus == .loaded,
```

Comment:
We typically don't check the load status. If the load errors out, the method will throw.